gslib/utils/shim_util.py

```diff
@@ -578,6 +592,12 @@ def translate_to_gcloud_storage_if_requested(self):
               ' You can make gsutil use the same credentials by running:\n'
               '{} config set pass_credentials_to_gsutil True'.format(
                   gcloud_binary_path))
+        elif (boto_util.UsingGsHmac and
```

Comment:
I think this should be boto_util.UsingGsHmac()

Reply:
Done.